test/unit_tests/graphql/test_product_blocks.py

```diff
@@ -107,6 +107,7 @@ def test_product_blocks_query(test_client):
             "inUseBy": [],
         },
     ]
+    product_blocks.sort(key=lambda x: x["name"])  # No sort in the query; sort before the assert to prevent flaky tests
```

Comment:
Mooier om sets te gebruiken bij de assert

Reply:
Not possible to hash dicts

Reply:
you can also use` key=itemgetter("name")`